pyk/src/pyk/cterm/symbolic.py

```diff
@@ -94,6 +96,28 @@ def kast_to_kore(self, kinner: KInner) -> Pattern:
     def kore_to_kast(self, pattern: Pattern) -> KInner:
         return kore_to_kast(self._definition, pattern)
 
+    def minimize_constraints(self, constraints: tuple[KInner, ...], path_condition: KInner) -> tuple[KInner, ...]:
+        """Minimize given branching constraints with respect to a given path condition."""
+        # By construction, this function is to be called with at least two sets of constraints
+        assert len(constraints) >= 2
+        # Determine intersection between all returned sets of branching constraints
+        flattened_default = tuple(flatten_label('#And', c) for c in constraints)
+        intersection = set.intersection(*(set(cs) for cs in flattened_default))
+        # If intersection is empty, there is nothing to be done
+        if not intersection:
+            return constraints
+        # Check if non-empty intersection is entailed by the path condition
+        dummy_config = self._definition.empty_config(sort=GENERATED_TOP_CELL)
+        path_condition_cterm = CTerm(dummy_config, constraints=[path_condition])
+        intersection_cterm = CTerm(dummy_config, constraints=intersection)
```

Comment:
Are constraints of a `CTerm` sorted? If not, they should be sorted before passing to the constructor.

Reply:
Yes, they are sorted.